### ISSUE TITLE
Not work

### ISSUE THREAD

**Issue by gekomod** [licensenulled/cPanel-Nulled#4]
it doesn't work, it stopped working after 5 days and it was supposed to be free.

**Comment by licensenulled** [comment]
yes, it works, you just have to activate it manually or pay a subscription. Servers and support must be maintained

**Comment by jesussuarz** [comment]
I see that the script is no longer available... :(

**Comment by licensenulled** [comment]
it is free for life, only it has to be activated every 5 days.
